symfony/framework-bundle/5.1/config/packages/prod/deprecations.yaml

```diff
@@ -0,0 +1,8 @@
+# Uncomment to log deprecations in a dedicated channel when using monolog-bundle
+#monolog:
```

Comment:
Why not keep this in the recipe of the Monolog-bundle?

As far as I can tell, this is not useable if the monolog-bundle is not installed.

Reply:
Because this works only when fwb 5.1 is installed, and we have no way to bind a recipe to two packages at the same time. I thought putting this in fwb is the best place, especially because it's commented.

Reply:
What about the other way around?

Put the comments in the monolog bundle and say "This only works on Framework bundle 5.1"

I'm fine with the changes. I let next reviewer give their opinion about this. Im 50/50.

Reply:
> But... that channels: [deprecation] config sorta ruins things

that's why I put this in a separate file yes.
I can put the file in the recipe for mobolog yes.

Reply:
updated,please check again